### DISCUSSION TITLE
SplitView.Pane not working on iOS

### DISCUSSION THREAD

**Discussion by MariusMK1** [AvaloniaUI/Avalonia#12772] [Q&A]
**Describe the bug**
A splitView.Pane is not working on iOS but works on desktop and android. When  pressing on a button it changes the value of the bound property in VM to true and it should open the pane but it does not open. But if you then rotate the device it opens it.

**To Reproduce**
<img width="862" alt="Screenshot 2023-09-02 at 17 08 23" src="https://github.com/AvaloniaUI/Avalonia/assets/93864647/d8fd49fc-844a-4393-af35-7680f1c9a666">

[ObservableProperty] private bool _menuOpen;
   
 [RelayCommand]
    private void MenuClick()
    {
        MenuOpen = true;
    }

**Expected behavior**
Should open Pane but it doesn't

- iOS any version
- Avalonia.iOS 11.0.4

**Comment by maxkatz6** [comment]
Do you use compiled bindings? Note, that if you use reflection bindings, properties/commands might be trimmed out. That's by design, as ios AOT trimmer is aggressive, and reflection bindings don't provide any type information.

https://docs.avaloniaui.net/docs/data-binding/compiled-bindings

**Reply by MariusMK1** [reply]
I'm using compiled bindings.
![image](https://github.com/AvaloniaUI/Avalonia/assets/93864647/ecb0205e-1bfa-4440-8001-19f956010945)
![image](https://github.com/AvaloniaUI/Avalonia/assets/93864647/899add7a-cb86-41ed-9442-10839d86e953)

**Reply by Amenti** [reply]
I can reproduce the same problem regardless of the used binding type. I also guess that the AOT trimmer is only active in release build. I can reproduce this behavior in debug build though.
